src/scss/_desktop.scss

```diff
@@ -30,9 +30,9 @@
             max-width: 1600px;
             width: 90%;
             margin: 5vh auto;
-            grid-template-columns: 1fr 1fr 1fr;
+            grid-template-columns: auto auto auto;
```

Comment:
has this been tested across all devices? Had problems with tablets doing it this way